extensions/positron-assistant/src/participants.ts

```diff
@@ -283,6 +283,9 @@ abstract class PositronAssistantParticipant implements IPositronAssistantPartici
 					// Only include the documentCreate tool in the chat pane in edit or agent mode.
 					case PositronAssistantToolName.DocumentCreate:
 						return inChatPane && (isEditMode || isAgentMode);
+					// Only include the getTableSummary tool for Python sessions until supported in R
+					case PositronAssistantToolName.GetTableSummary:
+						return positronContext.activeSession?.language.toLowerCase() === 'python';
```

Comment:
Ah, I think `activeSession` was refactored out of the context in https://github.com/posit-dev/positron/pull/8495.

This change works for me locally:

```suggestion
						// TODO: Remove this restriction when the tool is supported in R https://github.com/posit-dev/positron/issues/8343
						// The logic above with TOOL_TAG_REQUIRES_ACTIVE_SESSION will handle checking for active sessions once this is removed.
						return activeSessions.has('python');
```